ui/src/translations/de.ts

```diff
@@ -66,11 +66,11 @@ export const de = {
     unban_from_site: 'Von der Seite entbannen',
     banned: 'gesperrt',
     save: 'speichern',
-    unsave: 'unsave',
+    unsave: 'rückgängig schonen',
```

Comment:
Save ist speichern in dem Kontext. "Rückgängig machen" passt hier wohl ganz gut.

Reply:
Fixed!